src/fidesops/service/connectors/saas_query_config.py

```diff
@@ -87,6 +87,14 @@ def generate_requests(
                 request_params.append(
                     self.generate_query({string_path: [value]}, policy)
                 )
+
+        # Build SaaS requests for fields that are dependent on each other
+        grouped_inputs: List[Dict[str, Any]] = (
+            input_data.get(FIDESOPS_GROUPED_INPUTS) or []
```

Comment:
Is there a reason not to use `input_data.get(FIDESOPS_GROUPED_INPUTS, [])` here?

Reply:
no just missed in cleanup!